app/models/media_attachment.rb

```diff
@@ -407,6 +407,6 @@ def enqueue_processing
   end
 
   def reset_parent_cache
-    Rails.cache.delete("statuses/#{status_id}") if status_id.present?
+    Rails.cache.delete("v3:statuses/#{status_id}") if status_id.present?
```

Comment:
Instead of hard-coding in the string, could these either call `Status#cache_key` or reference a constant or something? Might avoid being missed on future changes.

Separately - tiny refactor idea would be to move the conditional onto the callback hook setup higher in the file -- `after_commit :reset_parent_cache, on: :update, if: :status_id?` or similar.

Reply:
Yeah, I considered doing that, but I wanted to avoid loading anything from the database or initializing an instance of `Status`